plugins/board-resources/src/components/popups/Member.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import { Employee, formatName } from '@anticrm/contact'
+  import { getFirstName, getLastName } from '@anticrm/contact'
+  import type { Ref } from '@anticrm/core'
+  import { IconCheck, Icon, Button } from '@anticrm/ui'
+
+  export let value: Employee
+  export let isSelected: boolean
+  export let onToggle: (member: Ref<Employee>) => Promise<void>
+
+  const firstName = getFirstName(value.name)
+  const lastName = getLastName(value.name)
+  const nameLabel = `${firstName?.[0] ?? ''}${lastName?.[0] ?? ''}`.toUpperCase()
+  const formattedName = formatName(value.name)
+</script>
+
+<div class="flex-row-center container" class:background-bg-accent={isSelected} on:click={() => onToggle(value._id)}>
+  <Button size="large" kind="no-border" shape="circle" title={formattedName}>
+    <div slot="content" class="text-md">{nameLabel}</div>
+  </Button>
```

Comment:
I think we should reuse `import { ContactPresenter } from '@anticrm/contact-resources'` here.

Also the name of the file/component is too ambiguous, which is not very good for code reading, please consider moving this code into `EditMembers` or renaming it.